crypto/cipher/aes_icm.c

```diff
@@ -92,7 +92,7 @@ srtp_debug_module_t srtp_mod_aes_icm = {
  *
  */
 
-static srtp_err_status_t srtp_aes_icm_alloc_ismacryp (srtp_cipher_t **c, int key_len, int forIsmacryp)
+static srtp_err_status_t srtp_aes_icm_alloc (srtp_cipher_t **c, int key_len)
```

Comment:
You are right, the signature should match. If you see other chnages in this commit you will see that the previous version of the function was `srtp_aes_icm_alloc (srtp_cipher_t **c, int key_len, int forIsmacryp)` not quit sure how tag_len should have mapped to forIsmacryp, but it was ignored anyway. I will add an extra argument.